tcms/testcases/tests/test_models.py

```diff
@@ -157,3 +158,28 @@ def test_send_mail_to_case_author(self, send_mail):
             recipients,
             fail_silently=False,
         )
+
+
+class TestCaseCalculateExpectedDuration(TestCase):
+    """Test TestCase.expected_duration"""
+
+    @parameterized.expand(
+        [
+            (
+                "both_values_are_set",
+                timezone.datetime(2021, 3, 22),
+                timezone.datetime(2021, 3, 23),
+                timezone.timedelta(days=1),
```

Comment:
This is still a verbatim copy of the previous test. It's not going to work because data types are different. Adjust accordingly.